services/auth.service.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
                 if (!!account) {
                     done(null, account);
                 } else {
-                    done("Invalid email or password.", false);
+                    done(null, false);
```

Comment:
What happens if we leave this message in?

Reply:
If we use the default error handler, we get an error 500, as the first argument is error. For our custom error handler, I've followed the same format, so we will also get a 500 error.

Reply:
Oh I see, so by putting in `Invalid email or password`, it doesn't call your custom callback?